src/sidebars/favourites/tearout.js

```diff
@@ -191,13 +189,11 @@
                                 }
                             };
 
-                            function reorderFavourites(tearoutRectangle) {
+                            function reorderFavourites() {
                                 var hoverTargets = hoverService.get();
 
                                 for (var i = 0, max = hoverTargets.length; i < max; i++) {
-                                    var dropTargetRectangle = geometryService.rectangle(
-                                        geometryService.elementScreenPosition(geometryService.getWindowPosition(window), hoverTargets[i].hoverArea)),
-                                        overDropTarget = tearoutRectangle.intersects(dropTargetRectangle);
+                                    var overDropTarget = geometryService.elementIntersect(tearoutWindow, window, hoverTargets[i].hoverArea);
```

Comment:
This and https://github.com/ScottLogic/bitflux-openfin/pull/440/files#diff-3709fe15a75397a1b30d1833b8c85403R211 are quite long lines, could you split them up?